src/server/terrain/terrain.zig

```diff
@@ -22,9 +24,85 @@ pub const StructureMap = @import("StructureMap.zig");
 
 pub const structure_building_blocks = @import("structure_building_blocks.zig");
 
+pub fn hashGeneric(input: anytype) u64 {
```

Comment:
This is only meant to be used by the biomes, so pelase keep it in biomes.zig for now.